packages/react-components/react-select/src/stories/SelectSize.stories.tsx

```diff
@@ -1,12 +1,29 @@
 import * as React from 'react';
 import { Select } from '../index';
+import { tokens } from '@fluentui/react-theme';
 import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@griffel/react';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: '400px',
+
+    '> * + label': { marginTop: tokens.spacingVerticalMNudge },
```

Comment:
Can we use `display: grid` and `gap` instead?

Reply:
I've actually changed this for SpinButton based on similar comments.

https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-spinbutton/src/stories/SpinButtonSize.stories.tsx#L8

Reply:
> Is there a benefit to grid over margin in this case?

The point of stories it to how consumers how to use components and let them copy-paste code. IMO `'> * + label'` as a selector is not something that I would recommend doing.

I would recommend to do:

```ts
const useStyles = makeStyles({
  root: { maxWidth: '400px' },
  label: { marginTop: tokens.spacingVerticalMNudge },
  // etc.
})
```

And avoid nested selectors if they are not needed.

Reply:
In a layout use case like this, modified versions of the lobotomized owl selector are actually exactly what I would recommend consumers use, at least if grid/flexbox weren't a good fit 😄. If I were writing styles that were specific to the labels themselves, I'd definitely add `label` to `makeStyles()`. This seems like a case where the parent should be able to handle its own layout for children without the children needing to modify themselves to participate.

Maybe a good approach to demo to consumers might be to import `selectClassNames.root` and use it here in place of the `*` selector, partly for marginally better selector performance, and partly to show consumers how to use it.

@spmonahan I have a very slight personal preference towards using `+` over using `:first-child { marginTop: 0 }`, but I'm also happy to switch to the latter. Was there any particular piece of feedback towards the latter?

Reply:
Please note following:

- Complex selectors harder to override due specificity
- CSS that comes from parent is harder to override (find parent first 🕹)
- The goal of Atomic CSS is to have less unique rules across an app: `.a { color: red }` & `.b * { color: red }` are different rules => more CSS output => more bundle size. It's one of the reasons to keep our styles flat